hugegraph-core/src/main/java/org/apache/hugegraph/structure/HugeElement.java

```diff
@@ -430,9 +430,6 @@ public static final ElementKeys classifyKeys(Object... keyValues) {
                 throw Element.Exceptions
                       .providedKeyValuesMustHaveALegalKeyOnEvenIndices();
             }
-            if (val == null) {
-                throw Property.Exceptions.propertyDoesNotExist();
-            }
```

Comment:
can you also fix issue 8: let throw Element.Exceptions.labelCanNotBeNull() if key.equals(T.label)